std/js/_std/HxOverrides.hx

```diff
@@ -146,6 +146,9 @@ class HxOverrides {
 		};
 	}
 
+	@:pure
+	static function now(): Float return untyped performance.now();
```

Comment:
Can we please not introduce any more `untyped`? :(

Reply:
I've replaced with `js.Browser.window.performance.now()`